CLOUD-715 add crVersion update (#30)
percona/k8spxc-docs: docs/update.md

    You can use `helm upgrade` to upgrade the Operator only. The Database Management System (Percona XtraDB Cluster) should be upgraded in the same way whether you used helm to install it or not.

## Upgrading Percona XtraDB Cluster

The following section presumes that you are upgrading your cluster within the
*Smart Update strategy*, when the Operator controls how the objects
are updated. Smart Update strategy is on when the `updateStrategy` key in the
[Custom Resource](operator.md) configuration file is set to `SmartUpdate`
(this is the default value and the recommended way for upgrades).

!!! note

>     As an alternative, the `updateStrategy` key can be used to turn off
>     *Smart Update strategy*. You can find out more on this in the
>     [appropriate section](update.md#more-on-upgrade-strategies).

Reviewer: ```suggestion
    As an alternative, the `updateStrategy` key can be set to `RollingUpdate` 
    and `OnDelete`. You can find out more about it in the
    [appropriate section](update.md#more-on-upgrade-strategies).
```

of the version. To check the availability of the new version, the Operator will
query a special *Version Service* server at scheduled times to obtain fresh
information about version numbers and valid image paths.

If the current version should be upgraded, the Operator updates the
Custom Resource to reflect the new image paths and carries on sequential Pods
deletion, allowing StatefulSet to redeploy the cluster Pods with the new image.
Pods are restarted by the Operator in the order, which assures the primary
instance to be updated last, preventing possible connection issues until the
whole cluster is updated to the new settings.

You can configure Percona XtraDB Cluster upgrade as follows:

1. First of all you should patch the Custom Resource version in the
    `spec.crVersion` field for your cluster to match the the Custom Resource
    Definition upgrade [you have done](update.md#manual-upgrade) while upgrading
    the Operator. The appropriate patch command for the cluster named `cluster1`
    looks as follows:

    ``` {.bash data-prompt="$" }
    $ kubectl patch pxc cluster1 --type=merge --patch '{"spec":{"crVersion":"{{ release }}"}}'
    ```

> 2. Set the `upgradeOptions.apply` option to one of the following values:

Reviewer: Also it might be cool to explain how to trigger the upgrade right away. Change the schedule to `* * * * *` ?

        (version numbers are specified as `{{ pxc80recommended }}`,
        `{{ pxc57recommended }}`, etc.),

>     * `Never` or `Disabled` - disable automatic upgrades (the default choice

Reviewer: remember, that this is the default
